ingeniamotion/wizard_tests/base_test.py

```diff
@@ -18,7 +18,12 @@ class TestError(Exception):
     pass
 
 
-class BaseTest(ABC, Stoppable):
+T = TypeVar("T")
+
+DictReportType = Dict[str, Union[SeverityLevel, Dict[str, Union[int, float, str]], str]]
```

Comment:
If it helps in anything typedict annotation exists from 3.8:
https://peps.python.org/pep-0589/
But maybe converting the reports to dataclasses would be a better action for the future...